plugins/services/src/js/utils/VipLabelUtil.js

```diff
@@ -29,6 +29,9 @@ const VipLabelUtil = {
   },
 
   findVip(labels = {}) {
+    if (typeof labels !== "object" || labels === null) {
+      return undefined;
```

Comment:
This is undefined, since this is the default for find. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find